### PR TITLE
Adding support for webservices transformations

### PR DESCRIPTION
**Pull request by ropalka**

### CODE REVIEW

**Comment by ropalka** [comment]
Please review @scottmarlow

File: impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties [hunk 1]
```diff
@@ -172,3 +172,9 @@ javax.xml.ws.spi=jakarta.xml.ws.spi
 javax.xml.ws.spi.http=jakarta.xml.ws.spi.http
 javax.xml.ws.wsaddressing=jakarta.xml.ws.wsaddressing
 
+
+# Namespaces Conversion
+http\://java.sun.com/xml/ns/javaee=https://jakarta.ee/xml/ns/jakartaee
+http\://java.sun.com/xml/ns/jaxb=https://jakarta.ee/xml/ns/jaxb
```
Comment (scottmarlow) [inline impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties:178]: Are we only parsing the new Jakarta namespace but not the old java.sun.com namespaces anymore?

Comment (ropalka) [reply impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties:178]: Do you think we should support also legacy namespaces @scottmarlow and not transform them?

Comment (scottmarlow) [reply impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties:178]: @ropalka yes, I think we should also support the legacy namespaces but I'm not against updating the older namespaces but I think we have to make sure the (SPEC) version referenced is correct I think.  

Typically, we have updated our schema parsing code to also parse the jakartaee namespaces in addition to the earlier namespaces.

http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/index.html#8 shows some of the older namespaces.

Comment (ropalka) [reply impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties:178]: I'll investigate tomorrow if we could support both, legacy and new namespaces.
The problem with webservices is it is not parsing DDs but instead it is using JAXB
with predefined bindings to un/marshall DDs.

Comment (scottmarlow) [reply impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties:178]: The minimum is that we support the Jakarta EE 8 schemas + Jakarta EE 9 schemas as per https://jakarta.ee/specifications/platform/9/jakarta-platform-spec-9.html#a3447,  We could talk with others on Web Services team about doing that versus also supporting earlier versions.  IMO, If we do cover the pre-Java EE 8 schemas, more coding is needed by someone.

Comment (ropalka) [reply impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties]: Hello @scottmarlow , please review this updated PR. I was able to eliminate namespaces transformations.

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
Thanks @ropalka looks good + merged! :-)

**Pull request merged**